### PR TITLE
MEPTS-1034 | MQ/MI CAT 7 Changes

### PR DESCRIPTION
**Pull request by iteliolucas**

### CODE REVIEW

File: api/src/test/resources/qualityImprovement2020.xml [hunk 1]
```diff
@@ -878,7 +884,13 @@ uuid="b534e130-da4c-42ef-811b-b94810112d4c"/>
 		date_created="2012-02-29 09:50:50" concept_name_id="2463" voided="0"
 		uuid="e262bc2c-1d5f-11e0-b929-000c29ad1d07"
 		locale_preferred="0" />
-	
+
+	<!-- 23984 -->
+	<concept concept_id="23984" retired="0" creator="1"
```
Comment (helderjosue) [inline api/src/test/resources/qualityImprovement2020.xml:889]: Please add a concept_name tag for this concept @TinoMuianga

Comment (TinoMuianga) [reply api/src/test/resources/qualityImprovement2020.xml:889]: Added the concept_name.

Comment (helderjosue) [reply api/src/test/resources/qualityImprovement2020.xml:889]: The **property_value** should not be changed, please revert the change for that part.
The change is to add the concept_name only. @TinoMuianga

Comment (TinoMuianga) [reply api/src/test/resources/qualityImprovement2020.xml:889]: The previous property_value was not right for the given concept, hence I have adjusted to the right one which is in the current implementation.

**Review by helderjosue — APPROVED** [review]
Looks good to me.

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Comment by ningosi** [comment]
@iteliolucas and @psmatsinhe this needs conflict resolution

**Pull request closed without merging**